plugin/src/main/java/io/jenkins/plugins/redis/RedisFingerprintStorage.java

```diff
@@ -153,12 +141,99 @@ public void delete(@NonNull String id) throws JedisException {
      * Returns true if there's some data in the fingerprint database.
      */
     public boolean isReady() {
-        try (Jedis jedis = getJedis()) {
+        JedisPoolManager jedisPoolManager = JedisPoolManager.INSTANCE;
+        try (Jedis jedis = jedisPoolManager.getJedis()) {
             return jedis.smembers(instanceId).size() != 0;
         } catch (JedisException e) {
             LOGGER.log(Level.WARNING, "Failed to connect to Jedis", e);
             throw e;
         }
     }
 
+    private String host = RedisFingerprintStorageDescriptor.DEFAULT_HOST;
```

Comment:
These should all be `final`

Reply:
These can't be `final` because these could be changed. It is the Jedis configuration.

Reply:
We discussed below b/c of the setters that are existing Jenkins plugin configuration pattern.